internal/sponge/sfix/sfix.go

```diff
@@ -0,0 +1,50 @@
+package sfix
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/ardanlabs/kit/db"
+	"github.com/coralproject/shelf/internal/sponge"
+	"github.com/coralproject/shelf/internal/sponge/item"
+	mgo "gopkg.in/mgo.v2"
```

Comment:
This may be a quirk of my setup, but when I remove mgo and save the file  the line is removed by go imports. My setup does infer package names in other places, but not here or for the Anvil code.